components/combobox/combobox.jsx

```diff
@@ -286,6 +288,9 @@ class Combobox extends React.Component {
 				(this.props.selection && this.props.selection[0]) || undefined,
 			activeSelectedOptionIndex: 0,
 		};
+
+		this.menuKeyBuffer = new KeyBuffer();
+		this.menuRef = React.createRef();
```

Comment:
We need to stick to 15.x API for a while still. Some clouds are still on React 15.x.